scripts/generate-meetup-data.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+var _ = require('lodash');
+var fs = require('fs');
+var targetFileName = 'free-code-camp-events.mock.json';
+var targetPath = 'js/';
+var meetup_api_key = _.get(process, 'env.MEETUP_API_KEY');
```

Comment:
> Wouldn't this be an easier set up for people to just place their key in a meetup_api_key.txt file that is in the .gitignore.

This is a valid alternative.

While I was creating the documentation I was indeed thinking these steps are a good introduction to learn more about shells and environment variables.  Perhaps I can add a "the easy way" and "the hard way" to the `README.md` to capture a pedagogical intent.

Unless you have objections, I'm going to build this out?  In the following code review we can then evaluate if we want both ways or just "the easy way."  :)